.buildkite/deploy.yml

```diff
@@ -0,0 +1,61 @@
+# Build and deploy to one of develop, staging, or production.
+#
+# Variables:
+#   VIRTRU_ENV: Environment to release to
+#   S3_PATH: path upload contents to on S3
+#   BLOCK_DEPLOY: [optional, defaults to true]
+
+env:
+  AWS_ACCOUNT_ALIAS: common
+  ENVIRONMENT_NAME: common
+  DEPLOY_ENVIRONMENT_NAME: production
+  S3_HOSTNAME: demos.developer.virtru.com
+  TEST_ENVIRONMENT_NAME: ${VIRTRU_ENV}
+  USE_NODE_12: true
+
+steps:
+  - label: 'annotate'
+    command: |
+      cat << EOF | buildkite-agent annotate --style "info"
+        [Triggered by ${BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG} Build #${BUILDKITE_TRIGGERED_FROM_BUILD_NUMBER}](/${BUILDKITE_ORGANIZATION_SLUG}/${BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG}/builds/${BUILDKITE_TRIGGERED_FROM_BUILD_NUMBER}#${BUILDKITE_TRIGGERED_FROM_BUILD_ID})
+      EOF
+    if: '"${BUILDKITE_SOURCE}" == "trigger_job"'
+
+  - label: ':house_with_garden: Create block for deploy environment selection'
+    branches: 'feature/* hotfix/*'
+    command: 'buildkite-scripts/mars/pipeline-steps/get-and-select-develop-environments.sh | buildkite-agent pipeline upload'
+
+  - wait
+
+  - label: ':house_with_garden: Set Environment'
+    command: 'buildkite-scripts/mars/scripts/set-deploy-environment.sh'
+    branches: 'master release/* feature/* hotfix/*'
```

Comment:
@dmihalcik-virtru with this are we going to be eliminating `develop` branch model per previous conversations?

Reply:
yeah it is already gone